docs/standard/collections/thread-safe/when-to-use-a-thread-safe-collection.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "When to Use a Thread-Safe Collection"
+description: Know when to use a thread-safe collection in .NET. Learn about 5 collection types that are specially designed to support multithreaded add & remove operations.
```

Comment:
```suggestion
description: Know when to use a thread-safe collection in .NET. There are 5 collection types that are specially designed to support multithreaded add & remove operations.
```

Reply:
Thanks. By the way, I did see your comment on the other PR about removing "Learn how to" or "Learn to". The Contributor's Guide guidance encourages us to include a "Call to action" (CTA), so that's why I added text like that. I'll try to vary the prose, or remove the CTAs if you want.

Reply:
I'm not sure either way. I think I added that comment but then realized I had already discussed it so I removed it 😄